### PR TITLE
feature/custom group

### PR DESCRIPTION
**Pull request by murtyk**

### CODE REVIEW

**Comment by murtyk** [comment]
@nikz ptal

**Comment by murtyk** [comment]
#83 
@nikz ptal we need to resolve this issue. Let me know if anything is pending that need to be addressed.

File: lib/raygun/client.rb [hunk 1]
```diff
@@ -141,6 +141,8 @@ def build_payload_hash(exception_instance, env = {})
         tags = env.delete(:tags) || []
         tags << rack_env if rack_env_present?
 
+        groupingKey = env.delete(:groupingKey)
```
Comment (nikz) [inline lib/raygun/client.rb]: Could you use `grouping_key` for this in the `ENV` hash and the local variable? We only want to change to `groupingKey` when we add it to the hash.

**Comment by murtyk** [comment]
#83 

@nikz ptal  changed the variable name to grouping_key.

**Comment by nikz** [comment]
Great thank you! Merged.

**Pull request merged**